.github/workflows/e2e-test.yml

```diff
@@ -78,7 +78,239 @@ jobs:
+        with:
+          php-version: ${{ matrix.php }}
+          tools: composer
+          extensions: mysql
+          coverage: none
+      # install dependencies
+      - run: composer self-update 2.0.6 && composer install --no-progress
+      - run: npm ci && npm run build:client
+      # prepare testing resources and run E2E tests
+      - run: echo -e "machine github.com\n  login $CI_USER_TOKEN" > ~/.netrc
+      - run: sudo systemctl start mysql.service
+      - run: WCPAY_DIR="$GITHUB_WORKSPACE" bash ./bin/install-wp-tests.sh woocommerce_test root root localhost $WP_VERSION $WC_VERSION false
+      - run: npm run test:e2e-setup
+      - run: E2E_GROUP='wcpay' E2E_BRANCH='shopper' npm run test:e2e
+      - uses: actions/upload-artifact@v2
+        with:
+          name: screenshots
+          path: screenshots
+  e2e-subscriptions-merchant:
+    name: Subscriptions | Merchant
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        # Min WooCommerce | L-2 | L(atest) | beta
+        woocommerce: [ '4.4.0', '5.7.0', '5.9.0', 'beta' ]
+        wordpress: [ 'latest' ]
+        php: [ '7.4' ]
+        include:
+          # Edge case: oldest dependencies compatibility
+          - woocommerce: '4.4.0'
+            wordpress: '5.4'
+            php: '7.0'
+    env:
+      WP_VERSION: ${{ matrix.wordpress }}
+      WC_VERSION: ${{ matrix.woocommerce }}
+    steps:
+      # clone the repository
+      - uses: actions/checkout@v2
+      # enable dependencies caching
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/composer/
+          key: ${{ runner.os }}-composer-${{ hashFiles('composer.lock') }}
+      - uses: actions/cache@v2
+        with:
+          path: vendor/
+          key: ${{ runner.os }}-vendor-${{ hashFiles('composer.lock') }}
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm/
+          key: ${{ runner.os }}-npm-${{ hashFiles('package-lock.json') }}
+      - uses: actions/cache@v2
+        with:
+          path: node_modules/
+          key: ${{ runner.os }}-node-modules-${{ hashFiles('package-lock.json') }}
+      # setup PHP, but without debug extensions for reasonable performance
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          tools: composer
+          extensions: mysql
+          coverage: none
+      # install dependencies
+      - run: composer self-update 2.0.6 && composer install --no-progress
+      - run: npm ci && npm run build:client
+      # prepare testing resources and run E2E tests
+      - run: echo -e "machine github.com\n  login $CI_USER_TOKEN" > ~/.netrc
+      - run: sudo systemctl start mysql.service
+      - run: WCPAY_DIR="$GITHUB_WORKSPACE" bash ./bin/install-wp-tests.sh woocommerce_test root root localhost $WP_VERSION $WC_VERSION false
+      - run: npm run test:e2e-setup
+      - run: E2E_GROUP='subscriptions' E2E_BRANCH='merchant' npm run test:e2e
```

Comment:
Same as above for variable names as constants.

Reply:
Done